core/src/main/java/com/example/AutomatedReviewTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 Coderanch.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.example;
+
+import java.util.*;
+import java.util.logging.*;
+
+import java.lang.*;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary import: java.lang classes are always implicitly imported. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack&pullRequest=9&issues=AWZEND_3vOM6zLh9bt9D&open=AWZEND_3vOM6zLh9bt9D)

[ISSUE_KEY]: AWZEND_3vOM6zLh9bt9D